packages/algob/src/types.ts

```diff
@@ -429,6 +429,9 @@ export interface AlgobDeployer {
   // Log Transaction
   log: (msg: string, obj: any) => void
 
+  // extract multi signed logic signature file from assets/
+  loadMultiSig: (name: string, scParams: Object) => Promise<Object>
+
   // get delegated Logic signature
   getDelegatedLsig: (lsigName: string) => Object | undefined
```

Comment:
this should also be updated, Please change the return type.

Reply:
fixed